src/main/java/io/appium/java_client/touch/RelativeOffsetOption.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.java_client.touch;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static java.util.Optional.ofNullable;
+
+import org.openqa.selenium.Point;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.internal.HasIdentity;
+
+import java.util.Map;
+
+public class RelativeOffsetOption extends ActionOptions<RelativeOffsetOption> {
+    private String elementId = null;
+    private Point relativeOffset = null;
+
+    /**
+     * It creates an instance of {@link RelativeOffsetOption } which takes an element
+     * and x and y offsets. These offsets are relative to the given element.
+     *
+     * @param element is the target element
+     * @param xOffset x-offset which is relative to the given element
+     * @param yOffset y-offset which is relative to the given element
+     * @return a built option
+     */
+    public static RelativeOffsetOption useRelative(WebElement element, int xOffset, int yOffset) {
+        return new RelativeOffsetOption().withRelativeOffset(element, xOffset, yOffset);
+    }
+
+    /**
+     *  * It creates an instance of {@link RelativeOffsetOption } which takes an element
+     * and 0-value x and y offsets.
+     *
+     * @param element is the target element
+     * @return a built option
+     */
+    public static RelativeOffsetOption useRelative(WebElement element) {
+        return useRelative(element, 0, 0);
```

Comment:
`0,0` should not be the case, but separate withElement should address this problem.